Commands/poll.py

```diff
@@ -0,0 +1,115 @@
+import discord
+import typing
+import asyncio
+import time
+
+from discord.ext import commands
+from Translations.Translations import get_text as translate
+from Helpers.TranslationHelper import get_culture_from_context as culture
+from Helpers.TranslationHelper import get_culture_from_context as get_culture_from_context
+from Helpers.emoji import number_emojis, yes, no, drum
+from Helpers.constants import POLL_MAX_TIMEOUT
+
+class Poll(commands.Cog, name="Simple Poll"):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+    
+    @commands.command(name="poll", brief="poll_brief", usage="poll_usage", help="poll_help")
+    async def poll(self, ctx: commands.Context, *, input_str: str):
+        """
+        Create a poll with either yes or no as an answer or self submitted options.
+        Poll will be open for an amount of time determined by the user.
+
+        Syntax: question (multiple words) timeout (numerals) <options> (words split by ;)
+        Example: This is a quention? 10 option 1; option 2; option 3
+
+        :param input: input to be parsed according to above syntax
+        """
+
+        poller_id = ctx.message.author.id
+
+        #check if there is a question
+        if '?' not in input_str:
+            await ctx.send(translate("poll_no_questionmark", await culture(ctx)))
+            return
+
+        input_split = input_str.split('?',1)
+        question = input_split[0]
+        numbers_and_options = input_split[1].strip()
+        
+        # parse timeout
+        first_word = numbers_and_options.split()[0]
+        if not first_word.isdigit():
```

Comment:
suggestion: why not default to POLL_MAX_TIMEOUT if no actual timeout is specified?

Reply:
Hmm, the MAX might be too long but I could revert it to a new POLL_DEFAULT_TIMEOUT and set that to 5 minutes. then send a message to the user informing him that it has been set to 5 minutes since he didn't supply it.

then similarly for when they supply a timeout bigger than POLL_MAX_TIMEOUT, set it to MAX_TIMEOUT and inform them.

Reply:
Sounds even better 👍